src/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+
+
+<html>
+
+	<!--contains about metadata, not displayed -->
+	<head>
+		<title> </title>
+		<meta charset="UTF-8"> 
+	</head>
+
+
+
+	<!-- contains the visible page content, only content here displayed -->
+	<body>
+
+		<!-- a large heading, the biggest font -->
+		<h1> </h1>
+		<h2> </h2>
+
+		<!-- separating contnet lines -->
+		<hr> 
+
+
+		<!-- Bullet points -->
+		<h3> An Unordered HTML List </h3>
+			<ul>
+				<li> Coffee </li>
+				<li> Tea </li>
+			</ul>
+
+		<!-- Numbered list -->
+		<h3> An Ordered HTML List </h3>
+			<ol>
+				<li> Coffee </li>
+				<li> Tea </li>
+			</ol>
```

Comment:
indentation is off here and in <ul> element before. It should be aligned with <h3> cuz it's not __inside__ of <h3>